Algorithms/Hashing/cuckoo_hashing.cpp

```diff
@@ -1 +1,88 @@
+// Name : Shivam Singhal
+// Email ID : shivamsinghal0610@gmail.com
 
+// Cuckoo hashing algorithm to improve worst case time for lookup in a table
+// based on multiple-choice hashing and relocation hashing
+
+#include <iostream>
+using namespace std;
+
+int indexTable[2];
+int n = 10;
```

Comment:
What this n stands for

Reply:
Alright

Reply:
@vikasgola I updated the file in my local repository but the change isn't reflecting here.

Reply:
Commit your changes and push it to remote.

Reply:
The PR says that it has been committed from an unknown repo.

Reply:
Did the changes reflected to your forked repo?

Reply:
Yes

Reply:
![screen shot 2018-07-22 at 2 26 07 pm](https://user-images.githubusercontent.com/24766191/43043990-36d59b02-8dbb-11e8-9de9-c1bedb1fe3f5.png)

@vikasgola

Reply:
@shivhek25 make a new pull request.

Reply:
@shivhek25 and also take n input from user instead of making it 10.